packages/ui/components/TemplateList/components/TemplateFilters.tsx

```diff
@@ -29,9 +29,8 @@ export const TemplateFilters = ({
       {allFilters.map((filters, index) => {
         if (prefilledFilters[index] !== null) return;
         return (
-          <div className="mt-2 flex flex-wrap gap-1 last:border-r-0">
+          <div key={index} className="mt-2 flex flex-wrap gap-1 last:border-r-0">
```

Comment:
Using an index as a key in React is generally considered bad practice because it can lead to performance issues and bugs, especially when the list changes dynamically (e.g., items are added, removed, or reordered). React relies on keys to efficiently update and re-render components, and using the index can cause incorrect behavior in these scenarios because the index does not uniquely identify the items.

Can we get some kind of other id from the filters?

Reply:
Thanks for pointing out this. I have updated it to `key={"key-" + filters[0]?.value || index}`. Now this code will use `filters` array first element's value as key and in case, `filters` array is empty, it will use index as backup key.

Hope this is okay? If anything else is required, please let me know.